src/main/java/uk/gov/hmcts/reform/sscs/service/SaveCorrespondenceAsyncService.java

```diff
@@ -25,9 +27,24 @@ public SaveCorrespondenceAsyncService(CcdNotificationsPdfService ccdNotification
         this.ccdNotificationsPdfService = ccdNotificationsPdfService;
     }
 
+    @Autowired
+    private LetterAsyncConfigProperties letterAsyncConfigProperties;
+
     @Async
-    @Retryable(maxAttemptsExpression = "#{${letterAsync.maxAttempts}}", backoff = @Backoff(delayExpression = "#{${letterAsync.delay}}", multiplierExpression = "#{${letterAsync.multiplier}}", random = true))
+    @Retryable(maxAttemptsExpression = "#{@letterAsyncConfigProperties.maxAttempts}", backoff = @Backoff(delayExpression = "#{@letterAsyncConfigProperties.delay}", multiplierExpression = "#{@letterAsyncConfigProperties.multiplier}", random = true, maxDelayExpression = "#{@letterAsyncConfigProperties.maxDelay}"))
     public void saveLetter(NotificationClient client, String notificationId, Correspondence correspondence, String ccdCaseId) throws NotificationClientException {
+
+        RetryContext context = RetrySynchronizationManager.getContext();
+        if (context != null && context.getRetryCount() == 0) {
```

Comment:
Making sure custom delay is added for on first try as 99% of first attempts fail.

Reply:
What happens if context is null, will it also try calling .getRetryCount and will this cause an error? Should the second part be a nested if statement?

Reply:
No, Java will ignore later statement as soon as first statement is `false` as `false && (true/false)` will be `false`